src/components/settings/SettingsLMSTab/UnsavedChangesModal.tsx

```diff
@@ -11,61 +10,62 @@ const UnsavedChangesModal = ({
   close,
   exitWithoutSaving,
   saveDraft,
-}: UnsavedChangesModalProps) => (
-  <ModalDialog
-    title={(
-      <FormattedMessage
-        id="adminPortal.settings.learningPlatformTab.unsavedChangesModal.cancelModalTitle"
-        defaultMessage="Cancel Modal"
-        description="Title for the cancel modal on learning platform tab"
-      />
-  )}
-    isOpen={isOpen}
-    onClose={close}
-    variant="default"
-  >
-    <ModalDialog.Header>
-      <ModalDialog.Title>
-        <FormattedMessage
-          id="adminPortal.settings.learningPlatformTab.unsavedChangesModal.exitConfigurationTitle"
-          defaultMessage="Exit configuration"
-          description="Title for exiting configuration modal dialog"
-        />
-      </ModalDialog.Title>
-    </ModalDialog.Header>
-    <ModalDialog.Body>
-      <FormattedMessage
-        id="adminPortal.settings.learningPlatformTab.unsavedChangesModal.saveDraftMessage"
-        defaultMessage="Your configuration data will be saved under your Learning Platform settings"
-        description="Message for saving draft in configuration modal dialog"
-      />
-    </ModalDialog.Body>
-    <ModalDialog.Footer>
-      <ActionRow>
-        <Button onClick={close} variant="outline-primary">
-          <FormattedMessage
-            id="adminPortal.settings.learningPlatformTab.unsavedChangesModal.cancelButton"
-            defaultMessage="Cancel"
-            description="Cancel button text in configuration modal dialog"
-          />
-        </Button>
-        <Button onClick={exitWithoutSaving} variant="outline-primary">
+}: UnsavedChangesModalProps) => {
+  const intl = useIntl();
+  return (
+    <ModalDialog
+      title={intl.formatMessage({
```

Comment:
[inform] Resolves a TypeScript bug, since `ModalDialog`'s `title` prop only accepts string, not a React `Element`.